extensions/justablock/gitpenguin.js

```diff
@@ -0,0 +1,205 @@
+// Name: Git Penguin
+// ID: GitPenguin
+// Description: Manage and get files from your GitHub repository.
+// By: GarboMuffin <https://scratch.mit.edu/users/justablock/>
+// Original: justablock
+class GitPenguin {
+    getInfo() {
+        return {
+            "id": "gitpenguin",
+            "name": "Git Penguin",
```

Comment:
What do penguins have to do with this exactly?

Reply:
because it was probably originally made to be added to penguinmod.